runners/spark/pom.xml

```diff
@@ -72,7 +72,7 @@
                     <parallel>none</parallel>
                     <failIfNoTests>true</failIfNoTests>
                     <dependenciesToScan>
-                      <dependency>org.apache.beam:java-sdk-all</dependency>
+                      <dependency>org.apache.beam:beam-sdks-java-core</dependency>
```

Comment:
CC: @amitsela just tweaking this typo while I'm futzing with the Maven config. It might cause a trivial conflict elsewhere.